docs/search/criteria_reference/createdat_criterion.md

```diff
@@ -4,11 +4,13 @@ The `CreatedAt` Search Criterion searches for products based on the date when th
 
 ## Arguments
 
-- `createdAt` - indicating the date that should be matched, provided as a `DateTimeInterface` object
+- `createdAt` (PHP), `created_at` (REST) - indicating the date that should be matched, provided as a `DateTimeInterface` object
```

Comment:
REST doesn't use an object as PHP does.

[The REST parser get a string and pass it to `DateTime` constructor](https://github.com/ibexa/product-catalog/blob/main/src/bundle/REST/Input/Parser/Query/Criterion/CreatedAt.php#L38). I don't know how to handle this. Maybe something like:

```suggestion
- `createdAt` (PHP), `created_at` (REST) - indicating the date that should be matched, provided as a `DateTimeInterface` object in PHP, or as a string acceptable by `DateTime` constructor in REST
```